components/locid/src/macros.rs

```diff
@@ -131,23 +131,23 @@ macro_rules! subtags_variant {
 /// ```
 /// use icu::locid::{langid, LanguageIdentifier};
 ///
-/// const DE_AT: LanguageIdentifier = langid!("de_at");
+/// const CA_ES_VALENCIA: LanguageIdentifier = langid!("ca_ES-valencia");
 ///
-/// let de_at: LanguageIdentifier = "de_at".parse().unwrap();
+/// let ca_es_valencia: LanguageIdentifier = "ca_es-valencia".parse().unwrap();
 ///
-/// assert_eq!(DE_AT, de_at);
+/// assert_eq!(CA_ES_VALENCIA, ca_es_valencia);
```

Comment:
I'd suggest reverting this change. We shouldn't overuse doc tests to show whole spectrum of capabilities - they should be more curated for common case well exemplified. Variants are outliers and most users of this API should not think about them so the main example shouldn't use them.

To test that it works, please add unit test at the bottom instead.

Reply:
Done